hobby-blog.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+    <link rel="stylesheet" type="css" href="stylesheets/styles_hobby.css"
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="hobby-blog.html">Photography</a></li>
+          <li><a href="about.html">About Me</a></li>
+        </ul>
+      </nav>
+      <h1>Photography</h1>
+    </header>
+
+    <article>
+      <h2 class="first-entry">Mystical Forest</h2>
+      <p>Kneeling on damp moss and decayed leaves, left cheek inches from the ground, squinting through the camera…I had a moment.  I was seeing the world in vivid detail—every shade of green on a velvety frond of moss, the shape a bug’s bite makes on a lacy, dry leaf, the stumbling yet determined path a termite takes on a pulpy, rotting tree trunk—details that I normally would’ve missed on a simple stroll through the forest.</p>
+      <p>...what else do I miss, everywhere, always?</p>
+      <div class="gallery">
+        <figure><a target="_new" href="https://www.flickr.com/photos/61965480@N03/24570241140/in/photostream/"><img alt= "a dead leaf eaten by bugs" src='images/a-dainty-eater.jpg'></a>
```

Comment:
`target="_blank"` is a safer choice here. `target="_new"` isn't included in the spec for the target attribute.